smoke-test/src/test/java/org/opennms/smoketest/rest/FileEditorIT.java

```diff
@@ -0,0 +1,336 @@
+package org.opennms.smoketest.rest;
```

Comment:
Need license header, can copy from another file. May sure to update copyright date to `1999-2022`